File: attribouter/src/main/java/me/jfenn/attribouter/data/github/LicenseData.java
package me.jfenn.attribouter.data.github;

public class LicenseData extends GitHubData {

    public String key;
    public String name;
    public String html_url;
    public String description;
    public String[] permissions;
    public String[] conditions;
    public String[] limitations;
    public String body;

    public LicenseData(String key) {
        super("https://api.github.com/licenses/" + key);
    }
}

File: attribouter/src/main/java/me/jfenn/attribouter/provider/net/github/data/LicenseData.kt
package me.jfenn.attribouter.provider.net.github.data

data class LicenseData(
        val key: String,
        val name: String?,
        val html_url: String?,
        val description: String?,
        val permissions: Array<String>?,
        val conditions: Array<String>?,
        val limitations: Array<String>?,
        val body: String?
)
